pkg/controller/config/config_controller.go

```diff
@@ -320,8 +322,14 @@ func (r *ReconcileConfig) applyAddons(req reconcile.Request, cfg *op.Config) (re
 	log := requestLogger(req, "apply-addons")
 
 	//add TaskProviderType label to ClusterTasks (community, redhat, certified)
-	injectLabel := transform.InjectLabel(flag.LabelProviderType, flag.ProviderTypeRedHat, transform.Overwrite, "ClusterTask")
-	if err := transformManifest(cfg, &r.addons, injectLabel); err != nil {
+	triggerImages := toLowerCaseKeys(imagesFromEnv("IMAGE_TRIGGERS_"))
+	addonImages := toLowerCaseKeys(imagesFromEnv("IMAGE_ADDONS_"))
```

Comment:
same comments for these 2
https://github.com/openshift/tektoncd-pipeline-operator/pull/251/files#r388725616